js/independentreserve.js

```diff
@@ -301,6 +324,44 @@ module.exports = class independentreserve extends Exchange {
         return this.parseOrder (response, market);
     }
 
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const pageIndex = this.safeInteger (params, 'pageIndex', 1);
+        if (limit === undefined) {
+            limit = 50;
+        }
+        const request = this.ordered ({});
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['primaryCurrencyCode'] = market['baseId'];
+            request['secondaryCurrencyCode'] = market['quoteId'];
+        }
+        request['pageIndex'] = pageIndex;
+        request['pageSize'] = limit;
```

Comment:
Does the exchange API require pageSize and pageIndex?
If they allow omitting these params, we should instead check if limit is not undefined and only set it if it is defined.
We don't need to set the pageIndex param if it is optional, since the override happens within the `.extend` in the next line 342.
What happens if you comment out line 341? Does it still work?